site/jest.config.js

```diff
@@ -1,28 +1,28 @@
-// REMARK: Jest is supposed to never exceed 50% maxWorkers by default. However,
-//         there seems to be an issue with this in our Ubuntu-based workspaces.
-//         If we don't limit it, then 100% CPU and high MEM usage is hit
-//         unexpectedly, leading to OOM kills.
-//
-// SEE thread: https://github.com/coder/coder/pull/483#discussion_r829636583
-const maxWorkers = 2
-
 module.exports = {
-  maxWorkers,
+  testTimeout: 10_000,
+  maxWorkers: 8,
```

Comment:
Isn't this value going to be set in CI anyways? It'd be ideal to use all 64 cores in dogfood

Reply:
Yes, but when it happens into 64 cores at the same time, the output is kinda crazy. I think we can remove it if it is too slow tho but with 8 workers, it is kinda fast.

Reply:
Makes sense!